app/cafe.py

```diff
@@ -0,0 +1,30 @@
+import datetime
+from app.errors import (NotVaccinatedError,
```

Comment:
According to the provided checklist, you should use `( )` while importing multiple classes/modules. Please, consider changing your import statement to follow the recommended style. Here is how it should look like:

python
from app.errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError
)
```